test/core/util/passthru_endpoint.h

```diff
@@ -24,6 +24,7 @@
 #include "src/core/lib/iomgr/endpoint.h"
 
 typedef struct {
```

Comment:
I did update all the places where grpc_passthru_endpoint_stats is used to use a pointer, but in theory nothing is preventing the struct to be used as a by-value field `grpc_passthru_endpoint_stats stats_` in new code (which is presumably not good because of the `refs`). Perhaps a better encapsulation would be useful.

Reply:
This is `test/core/util`, so a comment putting that restriction in place should be sufficient.

Reply:
sg.

Reply:
comment added.